requirements.txt

```diff
@@ -5,5 +5,7 @@ git+https://github.com/inducer/pymbolic.git#egg=pymbolic
 git+https://github.com/inducer/pyopencl.git#egg=pyopencl
 git+https://github.com/inducer/islpy.git#egg=islpy
 
-git+https://github.com/inducer/loopy.git#egg=loopy
-git+https://github.com/inducer/pytato.git#egg=pytato
+git+https://github.com/inducer/loopy@kernel_callables_v3-edit2.git#egg=loopy
+git+https://github.com/kaushikcfd/pytato.git@call_loopy#egg=pytato
+
+git+https://github.com/inducer/meshmode.git#egg=meshmode
```

Comment:
This goes the wrong way... what's the rationale for adding this?

Reply:
arraycontext/impl/pytato.py and arraycontext/impl/pyopencl.py use `meshmode.dof_array.DOFArray`

Reply:
If you're talking about this:
https://github.com/inducer/arraycontext/blob/23121b12f86faad1ce8cae2556b2f7f19b30fe18/arraycontext/impl/pyopencl.py#L175-L193
That's a deprecated code path that's only there for backward compatibility. It doesn't justify installing anything IMO.

Reply:
I don't see any other mention of `meshmode`.

Reply:
To provide more perspective: The assertion that this package works without meshmode is more valuable (to me) than the assertion that this busted code path works.

Reply:
I removed most of the meshmode code in https://github.com/inducer/arraycontext/pull/14/commits/e9156d5dc18e9fb2ca7648e49ecc896221e9fee5 and refactored the DofArray check in https://github.com/inducer/arraycontext/pull/14/commits/ef786c92b715e64a06e74f9b97e83b71700b3b85